### PR TITLE
Add aria-required to enable SR to read out required field

### LINKED ISSUE #8174: [Accessibility] [Narrator] Combo-Box control label readout incorrectly when required is true
### Environment Information 
 
- __Package version(s)__: 6.146.1
- __Browser and OS versions__: Chrome 72.0.3626.119

### Please provide a reproduction of the bug in a codepen:
 

Code: https://codepen.io/aniketkhorjuvekar/pen/JzKvzm?&editable=true
Narrator Used: NVDA

#### Actual behavior: 
Narrator transcript:
ComboBox with toggleable freeform/auto-complete *  combo box  collapsed  editable
 
#### Expected behavior:
 
Narrator transcript:
ComboBox with toggleable freeform/auto-complete  combo box  collapsed  editable required

'*' should be readout as "required" 

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by aftab-hassan**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #8174
- [ ] Include a change request file using `$ npm run change`

#### Description of changes
The ComboBox component was missing an `aria-required` and so, not reading out the `required` label, in case of empty input.

#### Focus areas to test
Create a ComboBox with `required={true}`. Use a screen reader technology like Narrator or NVDA and verify that the word **required** is read out

#### Screen Readers used to test
1. Narrator on Edge
2. NVDA on Google Chrome

#### Output
_Behavior before fix:_
ComboBox with toggleable freeform/auto-complete editable combo box collapsed

_Behavior after fix:_
ComboBox with toggleable freeform/auto-complete editable combo box **required** collapsed

### CODE REVIEW

**Comment by KevinTCoughlin** [comment]
Seeing some strange CI errors -- https://travis-ci.org/OfficeDev/office-ui-fabric-react/builds/508136103#L3929

Perhaps merge `master` if you haven't recently.

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.doc.tsx [hunk 1]
```diff
@@ -23,6 +24,9 @@ const ComboBoxVirtualizedExampleCodepen = require('!raw-loader!office-ui-fabric-
 const ComboBoxCustomStyledExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.CustomStyled.Example.tsx') as string;
 const ComboBoxCustomStyledExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/ComboBox/ComboBox.CustomStyled.Example.Codepen.txt') as string;
 
+const ComboBoxRequiredExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Required.Example.tsx') as string;
+const ComboBoxRequiredExampleCodepen = require('!raw-loader!office-ui-fabric-react/lib/codepen/components/ComboBox/ComboBox.Required.Example.Codepen.txt') as string;
```
Comment (aftab-hassan) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.doc.tsx]: @ecraig12345 would you have a recommendation on how to solve this?

I am adding a codepen for which I added the above line. This gives me a build error Error: Can't resolve 'office-ui-fabric-react/lib/codepen/components/ComboBox/ComboBox.Required.Example.Codepen.txt'. I pretty much went by the other examples, but do not see any of the other .txt files either and am not sure where to check these in.

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.doc.tsx]: Those files are autogenerated. There may be a list you have to update somewhere, @kenotron would know.

Comment (ecraig12345) [reply packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.doc.tsx]: The proper way to load codepen components has changed. Merge with master then look at one of the other lines that loads a codepen file to see the new way.

Comment (aftab-hassan) [reply packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.doc.tsx]: thanks @ecraig12345 and @natalieethell

File: packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Required.Example.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    requiredField: true
+  };
+
+  public render(): JSX.Element {
+    const state = this.state;
+    return (
+      <Fabric className={wrapperClassName}>
+        <ComboBox
+          label="ComboBox with required input"
+          key={'' + state.requiredField}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Required.Example.tsx]: Just curious, does `requiredField` state as the `key` value serve a purpose?

Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Required.Example.tsx]: This does seem odd

**Comment by KevinTCoughlin** [comment]
This change works well with Narrator given my testing. It now announces that the input is required when specified as such:

```shell
ComboBox with required input, read-only combo box,Required,
```

LGTM once remaining comments are resolved.

**Pull request merged**